Networking/Networking/Model/TaxClass.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+/// Represent a Tax Class Entity.
+///
+public struct TaxClass: Decodable {
+
+    /// Tax class name.
+    ///
+    public let name: String?
+
+    /// Unique identifier for the resource.
+    ///
+    public let slug: String?
```

Comment:
I feel it might make sense for these two properties (esp. `slug`) to be non-optional - from the [API doc](https://woocommerce.github.io/woocommerce-rest-api-docs/#tax-class-properties), it looks like `slug` is the unique identifier for a tax class. it probably makes sense to have this field always non-nil? it doesn't seem possible to assign a tax class to a product with a nil slug